### PR TITLE
ResetPassword

### PR DESCRIPTION
**Pull request by noobsi**
rubocop

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
```
Comment (ducbet) [inline db/schema.rb]: comment

File: db/schema.rb [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 
+ActiveRecord::Schema.define(version: 2018_08_06_085113) do
+
+  # These are extensions that must be enabled in order to support this database
```
Comment (ducbet) [inline db/schema.rb]: comment

**Comment by thaotonto** [comment]
ok

**Comment by ducbet** [comment]
ok

**Pull request merged**